### DISCUSSION TITLE
Trying to whitelist a new game (Flight of Nova) but it only sees vjoy devices

### DISCUSSION THREAD

**Discussion by B-Bonus** [nefarius/HidHide#94] [Q&A]
Hello - not sure if I am doing something wrong but whitelisting this app (Flight of Nova) doesn't seem to enable use of actual devices - any advice much appreciated. Thanks.

![HidHideClient_cKOt4VNpDj](https://user-images.githubusercontent.com/117114156/204929018-b436d61a-a54a-406e-aed3-a24ebb9e96d6.png)
![HidHideClient_z3flGNLUvc](https://user-images.githubusercontent.com/117114156/204929097-0238a291-b669-49f6-9578-94b171c7adb0.png)
![HidHideClient_G9tDbmn4yn](https://user-images.githubusercontent.com/117114156/204929104-064987d8-9ccb-4133-b2e0-a799f35eb908.png)
![FlightOfNova_eUcuv3CQZc](https://user-images.githubusercontent.com/117114156/204929117-654ba24a-0aae-4952-9d14-0f84ee2f7041.png)

Update - tried inverse cloak, only listed the two games I want to use with vJoy, but same result. Tested in another game and have the same issue, wondering if this is related to Steam.

Update 2 - had to restart a few times for other reasons and it's now working as expected. No idea why.

**Comment by enterthestream** [comment]
How long ago since you first installed it? Windows updates seem to mess with HidHide and reinstalling has fixed it for me in the past when it suddenly wouldn't work in a game. Looks like it worked itself out this time!

**Reply by B-Bonus** [reply]
Hi I first installed years ago, but reinstalled latest version whilst trying to fix this issue. Will bear in mind this tip in case of any issues in the future, many thanks.
